modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -98,9 +98,14 @@ export default class SolidPolygonLayer extends Layer {
           positions: {
             size: 3
           },
+          instancedPositions: {
+            size: 3,
+            divisor: 1
+          },
           nextPositions: {
             size: 3,
-            offset: 12
+            offset: 12,
+            divisor: 1
```

Comment:
I'm thinking, if we have to put something in this object instead of repeating `size`, we can instead draw attention to the difference between these attributes?

```js
positions: {divisor: 0, offset: 0},
instancedPositions: {divisor: 1, offset: 0},
nextPositions: {divisor: 1, offset: 12}
```

Reply:
> instead of repeating `size`, we can instead draw attention to the difference between these attributes?

+1, this would leverage the spirit of the luma.gl accessor object system, where accessor objects can be partial and are merged with defaults.

in luma.gl v6+, `size` is auto deduced and never needs to be specified (unless the app wants to use incomplete or too large attributes). I am not 100% sure this works in deck.gl without additional work as there are multiple layers of attribute handling code in play here, but we should strive towards this direction.